### PR TITLE
Fix some small issues in Home Page View and ViewModel

### PR DESCRIPTION
**Pull request by leeminkyu16**

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/home/HomePageViewModel.kt [hunk 1]
```diff
@@ -58,15 +51,28 @@ class HomePageViewModel @Inject constructor(
...
+		runBlocking {
+			api.listingsList(
+				priceMin = null,
+				priceMax = null,
+				roomsMin = null,
+				roomsMax = null,
+			)
+		}
+	}
+		.subscribeOn(Schedulers.io())
```
Comment (Jiay0928) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/home/HomePageViewModel.kt:76]: I am totally not sure, but is there any error handling for this API call?

Comment (leeminkyu16) [reply android/app/src/main/java/org/uwaterloo/subletr/pages/home/HomePageViewModel.kt:76]: Nope, I totally forgot to add it 😅 
Good catch, would have been pretty bad if we let this through

**Review by Jiay0928 — APPROVED** [review]

**Review by andr-dev — APPROVED** [review]
Nice work!

**Review by andr-dev — APPROVED** [review]
Rebase :upside_down_face:

**Pull request merged**